hr_timesheet_name_customer/models/hr_timesheet_name_customer.py

```diff
@@ -0,0 +1,14 @@
+from odoo import api, fields, models
+
+
+class NameCustomer(models.Model):
+    _inherit = "account.analytic.line"
+
+    name_customer = fields.Char()
+    """override create method, initialize name_customer"""
+
+    @api.model
+    def create(self, vals_list):
```

Comment:
Starting Odoo 12 you can create several record and once using create_multi decorator.
Check example [here](https://github.com/odoo/odoo/blob/2d42fd69cada3b1f2716c3d0a20bec6170f9b226/addons/account/models/account_account.py#L858)